docs/source/getting_started.md

```diff
@@ -1,14 +1,30 @@
 # Getting Started
-All instructions in this section assume you are using Ubuntu 22.04. The development dependencies should be installed when `dev_setup.sh` is run from the repository root.
+All instructions in this section assume you are using Ubuntu 22.04. The development dependencies should be installed when `source dev_setup.sh` is run from the repository root.
 
-## Using Docker
-We highly recommend using `docker` to containerize our code. This provides consistent environments and makes it easy to share code with others.
+There are two options for using `obelisk`. One is to use the same dependency management and containerization tools used for development to ensure reproducibility. The other is to simply install `obelisk` as a dependency of your project without using these tools. We will explain how to do both here.
 
-To build a minimal Docker image with only barebones ROS dependencies, run the following command from the `docker` directory:
+## Option 1: Isolation
+TODO(ahl): look this over and verify how to use isolation to do dependency management. Update docs in a separate PR.
+<!-- Once you have cloned this repository, you can run in the repo root:
 ```
-docker image build -t <image_name> -f Dockerfile.minimal .
+docker compose -f docker/docker-compose.yml run --build obelisk
 ```
-To enter the build container, run the following command in the repository root:
+This will load a Docker container with the repository root mounted into the container.
+
+If your downstream package is managed with `pixi`, you can install `obelisk` by [specifying the link to this repository](https://pixi.sh/latest/reference/project_configuration/#version-specification) in your `pixi.toml` file. -->
+
+## Option 2: Quick Installation
```

Comment:
Small nit-pick but I think we should put quick installation above isolation as I expect that is what most people will want.

Reply:
done